### ISSUE TITLE
HTML mode: Correctly scrapes but throws error on save/publish

### ISSUE THREAD

**Issue by cafterh** [Andful/netlify-cms-widget-create-select#5]
Error thrown: https://github.com/Andful/netlify-cms-widget-create-select/blob/487f39d441052af2683f16f7f4ff3eceac749580/src/Control.tsx#L96

Configuration
```yaml
- label: "Tags"
  name: "tags"
  hint: 'Do not duplicate any tags you would rather add for the whole Story!'
  widget: "create-select"
  mode: html
  url: FOO
  query: "h2.tag-title"
  attribute: "data-tag"
  multiple: true
  required: false
```

When this is used as a field, the html data is properly pulled and scraped into a list in the dropdown, but Save/Publish fails silently with an uncaught promise rejection, the field highlighting red and displaying the error above. Any assistance would be much appreciated. I was very excited to use this widget of yours.

**Comment by Andful** [comment]
This should be fixed by the commit 87df5cfa69ded123184670f00f581dc7ee36bbcb.

**Comment by Andful** [comment]
I published the library as version 1.1.3. Feel free to use it and let me know if there are any more bugs.

**Comment by cafterh** [comment]
Absolutely fantastic. I had made a whole workaround with the relation widget and a data file, and a task to update that list and....

This just works now. Cheers!!
